docs/py_docs.rst

```diff
@@ -1097,28 +1179,43 @@ For example:
       Requirements Document, LPM-17, URL https://ls.st/LPM-17
    """
 
+Attributes set in \_\_init\_\_ methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 In many classes, public attributes are set in the ``__init__`` method.
 The best way to document these public attributes is by declaring the attribute at the class level and including a docstring with that declaration:
 
 .. code-block:: python
+   :emphasize-lines: 12-18
 
    class Metric(object):
        """Verification metric.
 
-       [...]
+       Parameters
```

Comment:
might be good to have a parameter here in the example that is not also an attribute.